docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+django:
+  build: .
+  command: >
+    /bin/bash -c "
+    pip install -r requirements/dev.txt &&
+    python manage.py runserver 0.0.0.0:8001"
```

Comment:
Small change if we all agree, should we set the default port to 8002 ?
This is so that we can have the cashbook running at the same time as well.